Luciefifi/my-brand-backend-api
# Ft getallblogs swagger doc 025

#what does this PR do
add swagger documentation of getting all blogs

#description of the task to be completed
This PR adds documentation for the endpoint of getting all blogs in the API . the documentation includes the endpoint URL , the request method , the expected parameters , and the expected response
- install the dependencies to allow you document your APIs: swaggerjsdoc and swagger-ui-express
- create a file called documentations in helpers folder and add the definitions or optionds of your swagger document 
- create a documentations route
- create blog.doc.js file and add your get all blogs documentations
- import blog.doc.js file in documentations 
- localhost//5000:documentations
- - The endpoint is a GET request and expects no parameters
- The response is an object of blog objects, each containing the following properties: `id`, `title`, `description`, `image`, `blogBody` , "V"

#how should this be manually tested
after cloning the repository cd into it and run the server by npm run dev and then do the following
- Verify that the endpoint for getting all blogs is working as expected by making a GET request to `/api/getAllBlogs` and checking the response
- Verify that the new documentation in the Swagger documentation is accurate and matches the actual behavior of the endpoint
- localhost//5000:documentations

#relevant pivotal tracker  trello user story
#025
